RecoEgamma/EgammaTools/plugins/EGRegressionModifierV3.cc

```diff
@@ -63,8 +63,15 @@ class EGRegressionModifierV3 : public ModifyObjectValueBase {
   bool useClosestToCentreSeedCrysDef_;
   edm::ESHandle<CaloGeometry> caloGeomHandle_;
 
+  //because we're using a direct comp to this value to detect if its a default value
+  //we need to ensure its using the exact same type as its declared as
+  using TrkFBremType = decltype(reco::GsfElectron::ClassificationVariables::trackFbrem);
+  static const TrkFBremType kDefaultTrackFbrem_;
 };
 
+const EGRegressionModifierV3::TrkFBremType EGRegressionModifierV3::
+kDefaultTrackFbrem_ = reco::GsfElectron::ClassificationVariables().trackFbrem;
```

Comment:
this better be replaced by a direct (static const or constexpr) constant defined in ```GsfElectron::ClassificationVariables```.
I'm not sure why there should be an extra layer of creating a default object here.

Reply:
Done.